docs/source/aggregation.rst

```diff
@@ -0,0 +1,172 @@
+.. _aggregation:
+
+===========
+Aggregation
+===========
+
+This chapter provides information about the passing aggregates of jobs to operation functions.
+
+
+.. _aggregator_definition:
+
+aggregator
+==========
+
+An :class:`~flow.aggregator` is used as a decorator for an operation function which acts as an entry point to define the type of aggregation to perform while submitting or running the operation function.
```

Comment:
Could you try to rephrase the definition? I guess we don't have to be limited to one sentence, and I don't quite follow which "entry point" you're referring to. Can we also use it on multiple operation functions?

Reply:
Tried resolving here: https://github.com/glotzerlab/signac-docs/pull/122/files#diff-d053cd842dd6fe746188c4339762f7e336b763b94d513473d65900b9836a09f0R15